en/app.ftl

```diff
@@ -300,3 +300,17 @@ survey-option-not-likely = Not likely
 vpn-promo-headline = Save 50% with a full year subscription
 vpn-promo-copy = Protect your online data and choose a VPN subscription plan that works for you.
 vpn-promo-cta = Get { -brand-name-mozilla-vpn }
+
+## Email wrapping (header and footer messages wrapped around forwarded emails)
+
+# Variables:
+#   $display_email (string) - This is the relay email address. Example: abc123@relay.firefox.com
+#   $linked_origin (html) - This is a link to the website, wrapped with inline styles for email. Example: <a href="https://relay.firefox.com" ...>Firefox Relay</a>
+forwarded-email-header-from = This message was forwarded from { $display_email } by { $linked_origin }.
+# Variables:
+#   $faq_link (html) - This is a link to the website FAQ, wrapped with inline styles for email. Example: <a href="https://relay.firefox.com/faq" ...>{nav-faq}</a>
+forwarded-email-header-attachment = Firefox Relay supports email forwarding (including attachments) of email up to 150KB in size. To learn more check our { $faq_link }. 
```

Comment:
`Firefox Relay` is a brand. use the brand placeholder

Reply:
+1 to the `Firefox Relay`. We also have the 150KB as a variable too. See below:

```suggestion
forwarded-email-header-attachment = { -brand-name-firefox-relay } supports email forwarding (including attachments) of email up to { email-size-limit } in size. To learn more check our { $faq_link }. 
``` 

Note that you'll need to pass in two more variables to this string in the markup: 
`size='150' unit='KB'` (See `faq-question-7-answer` usage [for reference](https://github.com/mozilla/fx-private-relay/blob/main/privaterelay/templates/faq.html#L44))